multimodal/src/autogluon/multimodal/models/mmocr_text_detection.py

```diff
@@ -38,6 +38,8 @@ class MMOCRAutoModelForTextDetection(nn.Module):
     Refer to https://github.com/open-mmlab/mmocr
     """
 
+    data_types = [IMAGE]
```

Comment:
Data types for other OCR use cases could be [IMAGE, TEXT] too, will that be a problem in the future?

Reply:
Good point! I have removed this class attribute. Users can still customize the data_types for each model.